src/main/java/me/jellysquid/mods/sodium/client/gui/SodiumGameOptions.java

```diff
@@ -97,9 +98,9 @@ private static ChunkRendererBackendOption pickBestBackend() {
     }
 
     public enum GraphicsQuality implements TextProvider {
-        DEFAULT("Default"),
-        FANCY("Fancy"),
-        FAST("Fast");
+        DEFAULT(I18n.translate("generator.default")),
```

Comment:
Using `I18n#translate` here means that the translated strings will not change until the game is fully restarted, since the string will be cached and re-used. It should instead be moved into the TextProvider#getLocalizedName method below.

Reply:
I see.  Will fix then :+1: